x-pack/plugins/security_solution/common/endpoint/data_loaders/index_alerts.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Client } from '@elastic/elasticsearch';
+import { EndpointDocGenerator, Event, TreeOptions } from '../generate_data';
+import { firstNonNullValue } from '../models/ecs_safety_helpers';
+
+function delay(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+/**
+ * Indexes Alerts/Events into elasticsarch
+ *
+ * @param client
+ * @param eventIndex
+ * @param alertIndex
+ * @param generator
+ * @param numAlerts
+ * @param options
+ */
+export async function indexAlerts({
```

Comment:
I did not include a return value from this data indexer nor a `deleteIndexedAlerts()` because I'm not really familiar with how this loaded data is used (Detections, Event viewer?) If someone can give me some feedback here, I can add it and make it consistent like other data loaders.